extensions/amp-list/0.1/amp-list.js

```diff
@@ -557,7 +557,7 @@ export class AmpList extends AMP.BaseElement {
     const isSSR = this.ssrTemplateHelper_.isSupported();
     let renderPromise = this.ssrTemplateHelper_.renderTemplate(
         this.element, current.data)
-        .then(result => this.updateBindings_(result))
+        .then(result => this.updateBindings_(result, current.append))
```

Comment:
What type is `current` here?

Reply:
It's defined as
```
    this.renderItems_ = {data, append, resolver, rejecter,
      payload: opt_payload};
```

And typed here: 
```
     * @private {?{data:(?JsonObject|string|undefined|!Array), resolver:!Function, rejecter:!Function}}
```

Is it a bit cleaner this way? 
```
/** @typedef {{
  data:(?JsonObject|string|undefined|!Array),
  resolver:!Function,
  rejecter:!Function,
  append:boolean,
}} */
export let RenderItems;
```

Reply:
Thanks for looking, I was on my phone and couldn't find it.

The current type doesn't have an `append` (nor a `payload`). I'm not sure why this passes the type checker. We'll need to fix the type at least.

Reply:
Yup. Done. Yeah this is kind of not-very-intelligent code written to make templating things other than list-items work. I wish we could refactor this, but it's a little bit high risk.